### PR TITLE
Cloudflare integration announcement

### PR DESCRIPTION
**Pull request by SferaDev**
### Blog Release Checklist

- \[x] Release on: 14/12/23
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

File: cloudflare-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+## What is Cloudflare Workers?
+
+Cloudflare Workers is a serverless platform that allows developers to run code at the edge of the network. This means that you can run code closer to your users, which results in faster response times and lower latency. Cloudflare Workers also provide a secure environment for running code, so you don't have to worry about isolating your code from other users on the same server.
+
+## How does the integration look like?
+
+The integration is simple and straightforward, visit the Cloudflare dashboard and inside the "Integrations" tab, you will find the Xata integration. Click on the "Add integration" button and you will be redirected to the Xata dashboard where you can create a new database or select an existing one. Once you have selected the database, you will be redirected back to the Cloudflare dashboard and all the necessary environment variables will be automatically added to your worker.
+
+<ArticleImage
+  src="https://raw.githubusercontent.com/xataio/mdx-blog/cloudflare/images/cloudflare-integration-setup.png"
```
Comment (SferaDev) [inline cloudflare-integration.mdx:27]:
```suggestion
  src="https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cloudflare-integration-setup.png"
```

File: cloudflare-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: 'Xata + Cloudflare: One Click Database Integration with a Worker'
+description: 'Connecting a Cloudflare Worker with Xata is now possible from the Cloudflare dashboard.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-workers.png
```
Comment (SferaDev) [inline cloudflare-integration.mdx:5]: TODO

File: cloudflare-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  alt: Xata + Cloudflare
+author: Alexis Rico
+date: 12-12-2023
```
Comment (SferaDev) [inline cloudflare-integration.mdx:8]: Update

File: cloudflare-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  height="558"
+  alt="Cloudflare Integration Image"
+  caption="Wizard to connect your Cloudflare Worker with Xata"
+  position="center"
+/>
+
+During the process, you'll be prompted to authorize Cloudflare to access your Xata account. Once authorized, you can select the database you want to connect to your worker.
+
+<ArticleImage
+  src="https://raw.githubusercontent.com/xataio/mdx-blog/cloudflare/images/cloudflare-integration-authorization.png"
```
Comment (SferaDev) [inline cloudflare-integration.mdx:41]:
```suggestion
  src="https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cloudflare-integration-authorization.png"
```

File: cloudflare-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  height="613"
+  alt="Cloudflare Integration Image"
+  caption="Page to authorize Cloudflare to access your Xata account"
+  position="center"
+/>
+
+After authorizing Cloudflare, you'll be redirected back to the Cloudflare dashboard, where you can choose the database to connect to your worker.
+
+<ArticleImage
+  src="https://raw.githubusercontent.com/xataio/mdx-blog/cloudflare/images/cloudflare-integration-selection.png"
```
Comment (SferaDev) [inline cloudflare-integration.mdx:52]:
```suggestion
  src="https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cloudflare-integration-selection.png"
```

File: cloudflare-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+  height="843"
+  alt="Cloudflare Integration Image"
+  caption="Page to select the database you want to connect to your worker"
+  position="center"
+/>
+
+All necessary environment variables will be automatically added to your worker.
+
+<ArticleImage
+  src="https://raw.githubusercontent.com/xataio/mdx-blog/cloudflare/images/cloudflare-integration-secrets.png"
```
Comment (SferaDev) [inline cloudflare-integration.mdx:63]:
```suggestion
  src="https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cloudflare-integration-secrets.png"
```

File: cloudflare-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+published: true
+slug: cloudflare-integration
+---
+
+Cloudflare Workers, a serverless platform, empowers developers to run code at the edge of the network. This means running code closer to users, resulting in faster response times and lower latency. Cloudflare Workers also offer a secure environment, ensuring code isolation from other users on the same server.
+
+We are thrilled to announce a collaboration with Cloudflare, enabling a one-click integration directly from the Cloudflare dashboard. This user-friendly solution eliminates the need for developers to create an API key and manually configure their database connection.
+
+<ArticleImage
+  src="https://raw.githubusercontent.com/xataio/mdx-blog/cloudflare/images/cloudflare-integration-dashboard.png"
```
Comment (SferaDev) [inline cloudflare-integration.mdx:19]:
```suggestion
  src="https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cloudflare-integration-dashboard.png"
```

File: cloudflare-integration.mdx [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-workers.png
+  alt: Xata + Cloudflare
+author: Alexis Rico
+date: 12-12-2023
+tags: ['cloudflare', 'serverless', 'database', 'integration']
+published: true
+slug: cloudflare-integration
+---
+
+Cloudflare Workers, a serverless platform, empowers developers to run code at the edge of the network. This means running code closer to users, resulting in faster response times and lower latency. Cloudflare Workers also offer a secure environment, ensuring code isolation from other users on the same server.
```
Comment (eemmiillyy) [inline cloudflare-integration.mdx:14]:
```suggestion
Cloudflare Workers is a serverless platform that empowers developers to run code at the edge of the network. This means running code closer to users, resulting in faster response times and lower latency. Cloudflare Workers also offers a secure environment, ensuring code isolation from other users on the same server.
```

**Review by eemmiillyy — APPROVED** [review]

File: cloudflare-integration.mdx [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-workers.png
+  alt: Xata + Cloudflare
+author: Alexis Rico
+date: 12-12-2023
+tags: ['cloudflare', 'serverless', 'database', 'integration']
+published: true
+slug: cloudflare-integration
+---
+
+Cloudflare Workers is a serverless platform that empowers developers to run code at the edge of the network. This means running code closer to users, resulting in faster response times and lower latency. Cloudflare Workers also offers a secure environment, ensuring code isolation from other users on the same server.
```
Comment (alexfrancoeur) [inline cloudflare-integration.mdx:14]:
```suggestion
[Cloudflare Workers](https://developers.cloudflare.com/workers/) is a serverless platform that empowers developers to run code at the edge of the network. This means running code closer to users, resulting in faster response times and lower latency. Cloudflare Workers also offers a secure environment, ensuring code isolation from other users on the same server.
```

File: cloudflare-integration.mdx [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+<ArticleImage
+  src="https://raw.githubusercontent.com/xataio/mdx-blog/cloudflare/images/cloudflare-integration-dashboard.png"
+  width="874"
+  height="509"
+  alt="Cloudflare Integration Image"
+  caption="Cloudflare dashboard with the Xata integration"
+  position="center"
+/>
+
+Simply visit the Cloudflare dashboard, navigate to the "Integrations" tab, and find the Xata integration. Click on the "Add integration" button, and a wizard will guide you through the process of connecting your Cloudflare Worker with Xata.
```
Comment (alexfrancoeur) [inline cloudflare-integration.mdx:27]: This is a small nit, but I believe the integrations tab needs to enabled. I'm not sure if it's there by default.

https://developers.cloudflare.com/workers/databases/native-integrations/

File: cloudflare-integration.mdx [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+The integration is powered by an OAuth2 flow, allowing Cloudflare to access your Xata account and create a new API key for your worker. The API key is then used to connect your worker with your database.
+
+For more details on integrating with Xata using OAuth2, refer to our [documentation](/docs/integrations/oauth).
+
+## What's next?
+
+The integration is now available for all Cloudflare users. To learn more about how to use it, consult our [documentation](/docs/integrations/cloudflare).
+
+For questions or feedback, feel free to reach out to us on [Twitter](https://twitter.com/xata) or [Discord](https://xata.io/discord).
```
Comment (alexfrancoeur) [inline cloudflare-integration.mdx:106]:
```suggestion
For questions or feedback, feel free to reach out to us on [X / Twitter](https://twitter.com/xata) or [Discord](https://xata.io/discord). [Sign up](https://app.xata.io/signin?mode=signup) today to get started with Xata 🚀 
```

**Review by alexfrancoeur — APPROVED** [review]
Dropped in some minor suggestions but LGTM overall. Thanks for the quick turnaround @SferaDev !

File: cloudflare-integration.mdx (hunk 2 above)
Comment (SferaDev) [reply cloudflare-integration.mdx:5]: @alexfrancoeur We need an illustration

File: cloudflare-integration.mdx [hunk 12]
```diff
@@ -0,0 +1,108 @@
+---
+title: 'Connect your Cloudflare Worker with Xata in one click'
+description: 'Connecting a Cloudflare Worker with Xata is now possible from the Cloudflare dashboard.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-workers.png
+  alt: Xata + Cloudflare
+author: Alexis Rico
+date: 12-12-2023
```
Comment (SferaDev) [inline cloudflare-integration.mdx:8]:
```suggestion
date: 12-14-2023
```

File: cloudflare-integration.mdx [hunk 13]
```diff
@@ -0,0 +1,108 @@
+---
+title: 'Connect your Cloudflare Worker with Xata in one click'
+description: 'Connecting a Cloudflare Worker with Xata is now possible from the Cloudflare dashboard.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-workers.png
```
Comment (SferaDev) [inline cloudflare-integration.mdx:5]:
```suggestion
  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-cloudflare.png
```

File: cloudflare-integration.mdx [hunk 14]
```diff
@@ -0,0 +1,108 @@
+---
+title: 'Connect your Cloudflare Worker with Xata in one click'
+description: 'Connecting a Cloudflare Worker with Xata is now possible from the Cloudflare dashboard.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-cloudflare.png
+  alt: Xata + Cloudflare
+author: Alexis Rico
+date: 12-14-2023
+tags: ['cloudflare', 'serverless', 'database', 'integration']
+published: true
```
Comment (SferaDev) [inline cloudflare-integration.mdx:10]:
```suggestion
published: false
```

**Comment by SferaDev** [comment]
Merging with `published: false` for a final review

**Pull request merged**